source/reference/command/find.txt

```diff
@@ -74,6 +74,11 @@ The :dbcommand:`find` command has the following syntax:
       }
    )
 
+.. note::
+  Since MongoDB 4.4, the :dbcommand:`find` command ignores the
+  ``oplogReplay`` flag. However, if the ``oplogReplay`` flag is set, 
```

Comment:
[Minor wording suggestion]

I think we can omit "However" here in the interest of minimalist wording.

```suggestion
  ``oplogReplay`` flag. If the ``oplogReplay`` flag is set, 
```